.travis.yml

```diff
@@ -45,9 +45,12 @@ install:
   - make  # generate Python protobuf files and build shared libraries
 script:
   - tox
-  - poetry install -v
+  - poetry build -v
+  # export test dependencies from pyproject.toml, install them
+  - poetry export -f requirements.txt --dev -o requirements.txt && pip install -r requirements.txt
```

Comment:
Is this making the venv more isolated?

Reply:
What venv?

Reply:
When we first build the wheel with poetry, poetry handles the requirements for build (poetry creates a venv for this, right?).

Then we install runtime requirements with this line, again for the venv? Or is that `--dev` line doing something I don't understand.

After this we install the wheel to regular python, which is clean in our workflow so testing against it is valid way to do the testing.

Reply:
Oh, now I get it.

We don't have requirements.txt file, but poetry creates one with the help of `pyproject.toml`

And `--dev` exports [tool.poetry.dev-dependencies]

Reply:
That's right.